cadasta/gui/tools/wizard/step_project_creation03.py

```diff
@@ -257,18 +281,23 @@ def upload_parties(self):
             # Project is not uploaded
             return
 
-        network = NetworkMixin(get_url_instance() + post_url)
-
         for layer in self.data['locations']['features']:
             if layer['fields']['party_name'] and layer['fields']['party_type']:
                 post_data = QByteArray()
                 post_data.append('name=%s&' % layer['fields']['party_name'])
                 post_data.append('type=%s&' % layer['fields']['party_type'])
 
+                network = NetworkMixin(get_url_instance() + post_url)
```

Comment:
Shouldn't we rename `NetworkMixin` by something else ? We are not using the mixin in this case, we only use this object as usual OOP.